assets/css/main.css

```diff
@@ -1,714 +1,1100 @@
 html, body {
-  margin: 0;
-  padding: 0;
-  height: 100%;
-  font-family: helvetica, sans-serif;
-  line-height: 1.3;
-  color: #2b2b2b;
-  background: #FBFCFC;
-  font-size: 110%;
+    margin: 0;
+    padding: 0;
+    height: 100%;
+    font-family: "mr-eaves-xl-modern", helvetica;
+    font-weight:400;
+    line-height: 1.3;
+    color:#555;
+    background: #FBFCFC;
+    font-size: 110%;
 }
 
-a {
-  color: #F1853D;
-  text-decoration: none;
+div{
+    width:100%;
 }
 
-button {
-  background: #F1853D;
-  border: 0;
-  color: #fff;
-  padding: 13px 30px;
-  font-size: 1.35em;
+nav ul {
+    list-style: none;
 }
 
-input,textarea {
-  font-size: inherit;
-  font-family: inherit;
-  -webkit-appearance: none;
-  border-radius: 0;
+body {
+    font-size: 95%;
 }
 
-fieldset {
-  border: 0;
-  padding: 0;
-  margin: 0;
+input, select, textarea{
+    font-family: "mr-eaves-xl-modern", helvetica;
+    font-weight:400;
+}
+
+h1, h2, h3, h4, h5, h6{
+    font-family: "mr-eaves-xl-modern", helvetica;
+    font-weight:700;
+}
+h1{
+    font-size:64px;
+}
+h2{
+    font-size:48px;
+}
+h3{
+    font-size:48px;
+    color: rgb(24,34,41)
+}
+h4{
+    font-size:24px;
+}
+
+p{
+    font-size:18px;
+}
+
+a:hover{
+    cursor:pointer;
+}
+
+/*Top Bar*/
+
+.top-bar{
+    padding:20px 0 20px 0;
+    margin:0;
+    display:none;
+    position:fixed;
+    top:0;
+    left:0;
+    z-index:998;
+    background:rgb(24,34,41);
+    border-bottom:1px solid rgb(47,62,72);
+}
+.top-bar.after-fold{
+    display:block;
+    opacity:1;
+    -ms-filter: "progid:DXImageTransform.Microsoft.Alpha(Opacity=100)";
+}
+
+.top-bar .inner-wrapper{
+    max-width:1200px;
+    display:-webkit-box;
+    display:-ms-flexbox;
+    display:flex;
+    -webkit-box-pack: justify;
+    -ms-flex-pack: justify;
+    justify-content: space-between;
+    -webkit-box-align:center;
+    -ms-flex-align:center;
+    align-items:center;
+    margin:0 auto;
+    padding:0;
+    height:40px;
+}
+.logotype{
+    font-family: "mr-eaves-xl-modern", helvetica;
+    font-weight:400;
+    letter-spacing: 0.1em;
+    text-transform:uppercase;
+    text-align:left;
+    color:#fff1ef;
+    margin:0;
+    padding:0;
+}
+.logotype img{
+    text-align:left;
+    color:#fff1ef;
+    margin:0 15px 0 0;
+    padding:0;
+    font-size:24px;
+    width:300px;
+}
+.top-bar .inner-wrapper .cta{
+    text-align:right;
+    color:#fff1ef;
+    margin:0;
+    padding:0;
```

Comment:
Using a CSS reset [(like this one)](http://meyerweb.com/eric/thoughts/2011/01/03/reset-revisited/) could get rid of a lot of these margin/padding 0's but not necessary for merge.